lab_2_tokenize_by_bpe/main.py

```diff
@@ -47,6 +79,19 @@ def merge_tokens(
     :param pair: a pair of tokens to be merged
     :return: dictionary in the form of <preprocessed word: number of occurrences>
     """
+    if not isinstance(word_frequencies, dict) or not isinstance(pair, tuple):
+        return None
+    dict_with_tokens = {}
+    pair_indexes = f"'{pair[0]}', '{pair[1]}'"
+    for elem in word_frequencies:
+        list_with_words = list(elem)
+        if pair_indexes in str(elem):
+            for symbol in range(len(elem) - 1):
```

Comment:
This is a misleading variable name, it is not a symbol, right? It is the index of the symbol

Reply:
fixed